README.md

```diff
@@ -91,9 +91,10 @@ The TensorFlow project strives to abide by generally accepted best practices in
 
 ### Community Supported Builds
 
-| Build Type    | Status | Artifacts |
-| ---           | ---    | ---       |
-| **IBM s390x** | [![Build Status](http://ibmz-ci.osuosl.org/job/TensorFlow_IBMZ_CI/badge/icon)](http://ibmz-ci.osuosl.org/job/TensorFlow_IBMZ_CI/) | TBA |
+| Build Type      | Status | Artifacts |
+| ---             | ---    | ---       |
+| **IBM s390x**   | [![Build Status](http://ibmz-ci.osuosl.org/job/TensorFlow_IBMZ_CI/badge/icon)](http://ibmz-ci.osuosl.org/job/TensorFlow_IBMZ_CI/) | TBA |
+| **IBM ppc64le** | [![Build Status](http://powerci.osuosl.org/job/TensorFlow_Ubuntu_16.04_CPU/badge/icon)](http://powerci.osuosl.org/job/TensorFlow_Ubuntu_16.04_CPU/) | TBA |
```

Comment:
Do we want to say: IBM ppc64le CPU?

The IBM team is also working with OSU on the GPU side of things so that will be added later.

Thanks for adding this!

Reply:
We hope that `IBM ppc64le XLA` will be added later, too.

Reply:
Thanks for the information. Done.